Doc/library/stdtypes.rst

```diff
@@ -2289,10 +2574,18 @@ expression support in the :mod:`re` module).
    character(s) is not "Lu" (Letter, uppercase), but e.g. "Lt" (Letter,
    titlecase).
 
+   ::
+
+      >>> 'Monty Python'.upper()
+      'MONTY PYTHON'
+      >>> '𐊠'.upper().isupper()  # 'CARIAN LETTER A'
+      False
```

Comment:
```suggestion
      >>> '  '.upper().isupper()  # space only
      False
```

Reply:
I suggest adding the space only here, but keep the carian latter A.